tests/test_ffi_c_interpreter.nit

```diff
@@ -15,7 +15,7 @@
 import core::kernel
 
 fun foo `{ printf("FOO\n"); `}
-fun bar(i: Int) `{ printf("BAR %ld\n", i); `}
+fun bar(i: Int) `{ printf("BAR %ld\n", (long)i); `}
```

Comment:
It could be as well, but since this was a simple example I preferred to avoid the extra complexity of the macro, it is not really intuitive: `printf("%" PRIdFAST64 "", i)`